rllib/algorithms/algorithm.py

```diff
@@ -705,8 +705,11 @@ def setup(self, config: AlgorithmConfig) -> None:
             #  to be consistent with one another. To make a consistent parity between
             #  the two we need to loop through the policy modules and create a simple
             #  MARLModule from the RLModule within each policy.
-            local_worker = self.workers.local_worker()
-            module_spec = local_worker.marl_module_spec
+            module_spec = self.config.get_marl_module_spec(
```

Comment:
TODO: This might need polishing. It's not going to work for the other algos yet. We should figure out a unified and generic solution that does not depend on RolloutWorker "magic properties" :).

Reply:
I think getting the pure env from the RolloutWorker (vs DreamerV3EnvRunner) is the hacky part here. There is no clean API yet to access this object.
Actually, RolloutWorker also has a `self.env` property. Ok, this might actually work ...